src/components/homepage/SloganSection/index.tsx

```diff
@@ -2,25 +2,31 @@
  * Slogan section of homepage
  */
 
-import React from "react";
+import React, { useState } from "react";
 import Link from "@docusaurus/Link";
 import clsx from "clsx";
 import siteConfig from "@generated/docusaurus.config";
 
-import styles from "./index.module.scss";
 
 import WhiteHeighlinerSvg from "@site/static/img/homepage/white-heighliner.svg";
+import PurpleHeighlinerSVG from "/img/homepage/slogansection/purple-heighliner.svg"
```

Comment:
`"/img/homepage/slogansection/purple-heighliner.svg"`  -->   `@site/static/img/homepage/slogansection/purple-heighliner.svg` ,   keep  consistent and maintainable.